src/AccountableStatus.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace TestMonitor\Accountable;
+
+class AccountableStatus
+{
+    protected $enabled;
+
+    /**
+     * AccountableStatus constructor.
+     */
+    public function __construct()
+    {
+        $this->enabled = config('accountable.enabled');
+    }
+
+    /**
+     * @return bool
+     */
+    public function enable(): bool
+    {
+        return $this->toggleStatus(true);
+    }
+
+    /**
+     * @return bool
+     */
+    public function disable(): bool
+    {
+        return $this->toggleStatus(false);
+    }
+
+    /**
+     * @return bool
+     */
+    public function disabled(): bool
+    {
+        return $this->enabled === false;
+    }
+
+    /**
+     * @return bool
+     */
+    public function enabled(): bool
+    {
+        return $this->enabled;
+    }
+
+    /**
+     * @param $status
+     *
+     * @return bool
+     */
+    protected function toggleStatus($status)
```

Comment:
Deze snap ik niet helemaal. Deze class is een abstractie zodat je config niet hoeft te benaderen? Er staan hierboven methodes die die abstractie behouden, maar hieronder dan weer niet?

Toggle impliceert "ga van true naar false" en vice versa?